roxie/udplib/udpsim.cpp

```diff
@@ -154,6 +157,24 @@ void initOptions(int argc, const char **argv)
     udpTraceFlow = options->getPropBool("@udpTraceFlow", false);
     packetsPerThread = options->getPropInt("@packetsPerThread");
     udpTestUseUdpSockets = !options->getPropBool("@useQueue");
+    udpTestSocketDelay = options->getPropInt("@udpTestSocketDelay", 0);
+    udpTestSocketJitter = options->getPropBool("@udpTestSocketJitter", 0);
+    udpTestVariableDelay = options->getPropBool("@udpTestVariableDelay");
+    if (udpTestSocketDelay && udpTestUseUdpSockets)
```

Comment:
minor: last test needs inverting: && !udp...

Reply:
I don't think so - the option useQueue is the inverse of the boolean udpTestUseUdpSockets (because the latter defaults on)